rss2.xml

```diff
@@ -4,6 +4,7 @@
   xmlns:content="http://purl.org/rss/1.0/modules/content/">
   <channel>
     <title>{{ config.title }}</title>
+    <url>{{ icon }}</url>
```

Comment:
[The spec](http://www.rssboard.org/rss-specification) mentions `<url>` is required to be wrapped in `<image>`, and `<image>` must include `<title>`, `<link>` and `<url>`.

[This tutorial](https://www.xul.fr/en-xml-rss.html) also mentions `<image>`.

``` html
<title>{{ config.title }}</title>
<link>{{ url }}</link>
<image>
  <url>{{ icon }}</url>
  <title>{{ config.title }}</title>
  <link>{{ url }}</link>
</image>
```